soroban-sdk/src/testutils.rs

```diff
@@ -45,37 +45,6 @@ pub trait Logger {
     fn print(&self);
 }
 
-/// Test utilities for [`AccountId`][crate::accounts::AccountId].
-pub trait AccountId {
-    /// Generate a random account ID.
-    //
-    // The value filled is not cryptographically secure.
-    fn random(env: &Env) -> crate::AccountId;
-}
-
-/// Test utilities for [`Accounts`][crate::accounts::Accounts].
-pub trait Accounts {
-    /// Generate an account ID.
-    fn generate(&self) -> crate::AccountId;
```

Comment:
I assume we'd circle back on adding this before merging, but for testing there would need to be a way to generate distinct accounts / addresses that exist and don't exist, and to also use them for signing.

Reply:
Yeah, we may want this in some fashion, but for now I've just used `Account::random()`. I'm not sure we need more though; the regular contracts don't need to worry about the existence of an account and signatures are just not needed for the regular contract testing (which I think is a big win). Maybe wallet contracts would need more specific test utils; not sure about that yet as that's still WIP.